### ISSUE TITLE
Liste für Anwendungen auf Smartphone?

### ISSUE THREAD

**Issue by s72785** [fsfw-dresden/usb-live-linux#42]
Einfach mal eine Liste sammeln, was wir empfehlen würden; heute hat noch nicht jeder ein Notebook, aber ein Handy mit Adnroid/iOS ist sozusagen die Regel.

**Comment by gerdg-dd** [comment]
https://github.com/fsfw-dresden/usb-live-linux/blob/master/doc/src/android.md

**Comment by cknoll** [comment]
Ist ja schön und gut, aber wo ist der Zusammenhang zum Uni-Stick, also was macht das Issue in diesem Repo?

**Comment by s72785** [comment]
Zusammenhang ist die Liste für Windows, da kam auf auch auf Apps zu verweisen. zB LibreOffice, f-Droid...

**Comment by gerdg-dd** [comment]
so abwegig finde ich die Smartphone/Android Liste nicht - da Kommunikation z.B. Chat oder abgleichen von Adressen, Terminen, Aufgaben in einem  nahen Zusammenhang zu Programmen auf dem Stick stehen können. - Auswahl und Konfiguration wären noch zu diskutieren.

**Comment by cknoll** [comment]
Meinetwegen. (Nebenbemerkung: ich mag Smartphones nicht besonders... ;)).
Ich sehe nur (wie so oft), die Gefahr, dass wir zu viele Baustellen aufmachen und uns dann verzetteln. Ideen haben ist halt viel viel einfacher als sie umzusetzen.

**Comment by s72785** [comment]
Richtig, man muss auch nicht alles tun. Aber das kann man ja beschließen.
